insights/parsers/nova_log.py

```diff
@@ -0,0 +1,58 @@
+"""
+nova_log - files ``/var/log/nova/*.log``
+========================================
+This module contains classes to parse logs under ``/var/log/nova/``
+"""
+from .. import LogFileOutput, parser
+
+
+@parser('nova-api_log')
+class NovaApiLog(LogFileOutput):
+    """Class for parsing the ``/var/log/nova/nova-api.log`` file.
+
+    Note:
```

Comment:
Please modify the Note to sphinx style:
http://www.sphinx-doc.org/en/stable/markup/para.html#directive-note